src/routes/rss.xml.ts

```diff
@@ -0,0 +1,41 @@
+import rss from "@astrojs/rss";
+import { SITE_TITLE, SITE_DESCRIPTION } from "@src/consts";
+import { getLocale } from "i18n:astro";
+import { loadQuery } from "@api/sanity.ts";
+import type {
+  RssBlogPostsQueryResult,
+  SiteConfigQueryResult,
+} from "@sanity/sanity.types.ts";
+import { rssBlogPostsQuery, siteConfigQuery } from "@api/queries.ts";
+
+export async function GET(context) {
```

Comment:
This method isn't actually used it seems. I'm not sure how to test it either

Reply:
It is used. It for the rss feed. /en/rss.xml you can test it.